pkgs/tools/system/supercronic/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib, buildGoModule, fetchFromGitHub, python3, bash, coreutils }:
+
+buildGoModule rec {
+  pname = "supercronic";
+  version = "0.2.1";
+
+  excludedPackages = [ "cronexpr/cronexpr" ];
+  checkInputs = [ python3 bash coreutils ];
+  vendorHash = "sha256-oSwow6Ajh4HWrr+hfjlJIp1DOBCvN1Khm4tseWokxoc=";
```

Comment:
Could you put these attributes after `src`? It's common to have the `vendorHash` near the `src`'s hash - as they update together usually.